tests/distributed-transactions-test.py

```diff
@@ -100,7 +100,8 @@
         errorExit("Accounts creation failed.")
 
     Print("Spread funds and validate")
-    if not cluster.spreadFundsAndValidate(10):
+    # if activateIF then irreversible node needs funds to be irreversible before validation
+    if not cluster.spreadFundsAndValidate(10, waitForFinalization=activateIF):
```

Comment:
so if activeateIF, will always wait for finalization even in head and speculative modes?

Reply:
Yes